presto-hive/src/main/java/com/facebook/presto/hive/metastore/HiveColumnStatistics.java

```diff
@@ -38,16 +41,17 @@
     private final OptionalLong nullsCount;
     private final OptionalLong distinctValuesCount;
 
+    @JsonCreator
     public HiveColumnStatistics(
-            Optional<IntegerStatistics> integerStatistics,
-            Optional<DoubleStatistics> doubleStatistics,
-            Optional<DecimalStatistics> decimalStatistics,
-            Optional<DateStatistics> dateStatistics,
-            Optional<BooleanStatistics> booleanStatistics,
-            OptionalLong maxColumnLength,
-            OptionalDouble averageColumnLength,
-            OptionalLong nullsCount,
-            OptionalLong distinctValuesCount)
+            @JsonProperty("integerStatistics") Optional<IntegerStatistics> integerStatistics,
+            @JsonProperty("doubleStatistics") Optional<DoubleStatistics> doubleStatistics,
+            @JsonProperty("decimalStatistics") Optional<DecimalStatistics> decimalStatistics,
+            @JsonProperty("dateStatistics") Optional<DateStatistics> dateStatistics,
+            @JsonProperty("booleanStatistics") Optional<BooleanStatistics> booleanStatistics,
+            @JsonProperty("maxColumnLength") OptionalLong maxColumnLength,
```

Comment:
Same comment as in other commit, it seems that these should be a separate `StringStatistics` class